modules/objectdb_profiler/editor/data_viewers/refcounted_view.cpp

```diff
@@ -0,0 +1,320 @@
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#include "refcounted_view.h"
+
+#include "editor/editor_node.h"
+#include "editor/themes/editor_scale.h"
+#include "scene/gui/rich_text_label.h"
+#include "scene/gui/split_container.h"
+
+SnapshotRefCountedView::SnapshotRefCountedView() {
+	set_name(TTRC("RefCounted"));
+}
+
+void SnapshotRefCountedView::show_snapshot(GameStateSnapshot *p_data, GameStateSnapshot *p_diff_data) {
+	SnapshotView::show_snapshot(p_data, p_diff_data);
+
+	item_data_map.clear();
+	data_item_map.clear();
+
+	set_v_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+	set_h_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+
+	refs_view = memnew(HSplitContainer);
+	add_child(refs_view);
+	refs_view->set_anchors_preset(LayoutPreset::PRESET_FULL_RECT);
+
+	VBoxContainer *refs_column = memnew(VBoxContainer);
+	refs_column->set_anchors_preset(LayoutPreset::PRESET_FULL_RECT);
+	refs_view->add_child(refs_column);
+
+	// Tree of Refs.
+	refs_list = memnew(Tree);
+
+	filter_bar = memnew(TreeSortAndFilterBar(refs_list, TTRC("Filter RefCounteds")));
+	refs_column->add_child(filter_bar);
+	int offset = diff_data ? 1 : 0;
+	if (diff_data) {
+		filter_bar->add_sort_option(TTRC("Snapshot"), TreeSortAndFilterBar::SortType::ALPHA_SORT, 0);
+	}
+	filter_bar->add_sort_option(TTRC("Class"), TreeSortAndFilterBar::SortType::ALPHA_SORT, offset + 0);
+	filter_bar->add_sort_option(TTRC("Name"), TreeSortAndFilterBar::SortType::ALPHA_SORT, offset + 1);
+	TreeSortAndFilterBar::SortOptionIndexes default_sort = filter_bar->add_sort_option(
+			TTRC("Native Refs"),
+			TreeSortAndFilterBar::SortType::NUMERIC_SORT,
+			offset + 2);
+	filter_bar->add_sort_option(TTRC("ObjectDB Refs"), TreeSortAndFilterBar::SortType::NUMERIC_SORT, offset + 3);
+	filter_bar->add_sort_option(TTRC("Total Refs"), TreeSortAndFilterBar::SortType::NUMERIC_SORT, offset + 4);
+	filter_bar->add_sort_option(TTRC("ObjectDB Cycles"), TreeSortAndFilterBar::SortType::NUMERIC_SORT, offset + 5);
+
+	refs_list->set_select_mode(Tree::SelectMode::SELECT_ROW);
+	refs_list->set_custom_minimum_size(Size2(200, 0) * EDSCALE);
+	refs_list->set_hide_folding(false);
+	refs_column->add_child(refs_list);
+	refs_list->set_hide_root(true);
+	refs_list->set_columns(diff_data ? 7 : 6);
+	refs_list->set_column_titles_visible(true);
+
+	if (diff_data) {
+		refs_list->set_column_title(0, TTRC("Snapshot"));
+		refs_list->set_column_expand(0, false);
+		refs_list->set_column_title_tooltip_text(0, "A: " + snapshot_data->name + ", B: " + diff_data->name);
+	}
+
+	refs_list->set_column_title(offset + 0, TTRC("Class"));
+	refs_list->set_column_expand(offset + 0, true);
+	refs_list->set_column_title_tooltip_text(offset + 0, TTRC("Object's class"));
+	refs_list->set_auto_translate_mode(AUTO_TRANSLATE_MODE_DISABLED);
```

Comment:
This disables translations for the whole node, including column names. You should only apply it to TreeItems.

Reply:
Not sure what happened there, but that was my bad. In my mind I was probably passing in the column number there or something.